bin/ungit

```diff
@@ -90,7 +85,6 @@ server.listen(config.port, config.ungitBindIp, (err) => {
 server.on('error', (e) => {
   if (e.code == 'EADDRINUSE') {
     console.log('Ungit server already running');
-    navigate(true);
-    cleanExit();
```

Comment:
`cleanExit` prevented the browser from beeing launched on my machine (windows) because `open` now returns a promise. Letting node decide when to exit the process worked for me.
Could also return the promise from `navigate` but that would require more code.